qa/acceptance-tests/src/test/java/io/camunda/it/migration/IdentityDefaultEntitiesIT.java

```diff
@@ -10,53 +10,76 @@
-import org.testcontainers.elasticsearch.ElasticsearchContainer;
+import org.testcontainers.containers.wait.strategy.HttpWaitStrategy;
 import org.testcontainers.junit.jupiter.Container;
 import org.testcontainers.junit.jupiter.Testcontainers;
 
-@ZeebeIntegration
+@MultiDbTest
 @Testcontainers(parallel = true)
-@Disabled
 public class IdentityDefaultEntitiesIT {
 
-  @Container
-  private static final GenericContainer<?> POSTGRES = IdentityMigrationTestUtil.getPostgres();
+  @MultiDbTestApplication(managedLifecycle = false)
+  static final TestCamundaApplication camunda =
+      new TestCamundaApplication()
+          .withSecurityConfig(cfg -> cfg.getAuthorizations().setEnabled(true));
 
-  @Container
-  private static final GenericContainer<?> KEYCLOAK =
-      IdentityMigrationTestUtil.getKeycloak(POSTGRES);
+  static CamundaClient client;
 
   @Container
-  private static final GenericContainer<?> IDENTITY =
-      IdentityMigrationTestUtil.getManagementIdentity(POSTGRES, KEYCLOAK);
+  private static final KeycloakContainer KEYCLOAK = IdentityMigrationTestUtil.getKeycloak();
 
   @Container
-  private static final ElasticsearchContainer ELASTICSEARCH =
-      TestSearchContainers.createDefeaultElasticsearchContainer();
-
-  @TestZeebe(autoStart = false)
-  final TestCamundaApplication camunda =
-      new TestCamundaApplication()
-          .withSecurityConfig(cfg -> cfg.getAuthorizations().setEnabled(true));
-
-  @Test
-  void canMigrateOldDefaultEntities() {
+  private static final GenericContainer<?> IDENTITY =
+      IdentityMigrationTestUtil.getManagementIdentity(KEYCLOAK)
+          // create groups
+          .withEnv("KEYCLOAK_GROUPS_0_NAME", "groupA")
+          .withEnv("KEYCLOAK_GROUPS_1_NAME", "groupB")
+          .withEnv("KEYCLOAK_GROUPS_2_NAME", "groupC")
+          // create users and assign them to groups
+          .withEnv("KEYCLOAK_USERS_0_EMAIL", "user0@email.com")
+          .withEnv("KEYCLOAK_USERS_0_FIRST-NAME", "user0")
+          .withEnv("KEYCLOAK_USERS_0_LAST-NAME", "user0")
+          .withEnv("KEYCLOAK_USERS_0_USERNAME", "user0")
+          .withEnv("KEYCLOAK_USERS_0_PASSWORD", "password")
+          .withEnv("KEYCLOAK_USERS_0_GROUPS_0", "groupA")
+          .withEnv("KEYCLOAK_USERS_0_GROUPS_1", "groupB")
+          .withEnv("KEYCLOAK_USERS_1_EMAIL", "user1@email.com")
+          .withEnv("KEYCLOAK_USERS_1_FIRST-NAME", "user1")
+          .withEnv("KEYCLOAK_USERS_1_LAST-NAME", "user1")
+          .withEnv("KEYCLOAK_USERS_1_USERNAME", "user1")
+          .withEnv("KEYCLOAK_USERS_1_PASSWORD", "password")
+          .withEnv("KEYCLOAK_USERS_1_GROUPS_0", "groupC")
+          .waitingFor(
+              new HttpWaitStrategy()
+                  .forPort(8082)
+                  .forPath("/actuator/health/readiness")
+                  .allowInsecure()
+                  .forStatusCode(200)
+                  .withReadTimeout(Duration.ofSeconds(10))
+                  .withStartupTimeout(Duration.ofMinutes(5)));
```

Comment:
🔧 better we move that to `IdentityMigrationTestUtil` ? or does the testcontainers api not allow that?

Reply:
I don't want to move all the groups and users setup into the util class, that's why this waiting is placed here